modules/CONT-CLI-11_Viewing-local-diffs.yml

```diff
@@ -16,7 +16,7 @@ screens:
         - do: "Type `git diff --staged`"
           say: "When you add the --staged option, git diff will compare the staged version with the most recent committed version of the file."
         - do: "Type `git diff HEAD`"
-          say: "When you add HEAD after the git diff command, git combines the changes in your working and staged directories and compares them with the version of the file that is currently designated as the HEAD. In most cases, this is the most recent commit in the history of the file."
+          say: "When you add HEAD after the git diff command, git combines the changes in your working and staging areas and compares them with the version of the file that is currently designated as the HEAD. In most cases, this is the most recent commit in the history of the file."
```

Comment:
This is a nice accuracy update.  Thinking of little details, should git diff be `git diff` (wrapped in back ticks?)